src/Config/ConfigPanel.ts

```diff
@@ -0,0 +1,82 @@
+import * as vscode from 'vscode';
+import {getNonce} from './GetNonce';
+
+
+export class ConfigPanel {
+  /**
+   * Track the current panel. Only allow a single panel to exist at a time.
+   */
+  public static currentPanel: ConfigPanel|undefined;
+
+  public static readonly viewType = 'one-vscode';
+
+  private readonly _panel: vscode.WebviewPanel;
+  private readonly _extensionUri: vscode.Uri;
+  private _disposables: vscode.Disposable[] = [];
```

Comment:
Just a Q) members starting with `_` and without `_`, can you share where this convention came from?

Reply:
It came from vscode-extension-webview sample. and i make this panel by copying #25.
i saw _ things are used other place. i guess use _ to prevent confusing.

Reply:
> It came from vscode-extension-webview sample. and i make this panel by copying #25.

@mhs4670go , do we need to revise copyright message ?

Ah, there is no copyright message in this file :)